gridgencore.py

```diff
@@ -473,7 +473,15 @@ def preprocess(self):
             else:
                 self.total_run += 1
                 step_count = set.params.get("steps")
-                self.total_steps += int(step_count) if step_count is not None else self.p.steps
+                step_count = int(step_count) if step_count is not None else self.p.steps
+                self.total_steps += step_count
+                enable_hr = set.params.get("enable highres fix")
```

Comment:
This logic shouldn't be in this file, `gridgencore.py` is intentionally independent of Auto WebUI (such that it can be reused for any other python-based image gen tool)